src/style.css

```diff
@@ -0,0 +1,7 @@
+.row {
```

Comment:
This file is a copy+paste relict which can be removed.

Reply:
OK